`cloudflare_access_group` doesn't include `github` in `require` or `include` blocks in `PUT` request

### Confirmation

- [X] My issue isn't already found on the issue tracker.
- [X] I have replicated my issue using the latest version of the provider and it is still present.

### Terraform and Cloudflare provider version

```bash
terraform -v
Terraform v1.1.9
on darwin_arm64
+ provider registry.terraform.io/cloudflare/cloudflare v3.13.0
```

### Affected resource(s)

- `cloudflare_access_group`
- `cloudflare_access_identity_provider` (?)

### Terraform configuration files

```hcl
resource "cloudflare_access_group" "github_eragon_xyz_org_members" {
  account_id = var.account_id
  name       = "github-org-members"

  include {
    github {
      name                 = "github-org"
      identity_provider_id = cloudflare_access_identity_provider.github_oauth.id
    }
  }

  require {
    github {
      name                 = "github-org"
      identity_provider_id = cloudflare_access_identity_provider.github_oauth.id
    }
  }
}
```


### Debug output

```
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # cloudflare_access_group.github_eragon_xyz_org_members will be updated in-place
  ~ resource "cloudflare_access_group" "github_eragon_xyz_org_members" {
        id         = "0164a4ca-2bc2-415d-aad4-adf81ff345bf"
        name       = "github-eragon-xyz-org-members"
        # (1 unchanged attribute hidden)

      + include {

          + github {
              + identity_provider_id = "1541f135-e878-450f-816f-be39d33e18c5"
              + name                 = "eragon-xyz"
            }
        }

      + require {

          + github {
              + identity_provider_id = "1541f135-e878-450f-816f-be39d33e18c5"
              + name                 = "eragon-xyz"
            }
        }
    }

Plan: 0 to add, 1 to change, 0 to destroy.
2022-04-24T17:05:21.784-0700 [DEBUG] command: asking for input: "\nDo you want to perform these actions in workspace \"eragon\"?"

Do you want to perform these actions in workspace "eragon"?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

2022-04-24T17:05:28.150-0700 [INFO]  backend/local: apply calling Apply
2022-04-24T17:05:28.150-0700 [DEBUG] Building and walking apply graph for NormalMode plan
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "cloudflare_record.postgres_eragon_xyz (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "cloudflare_access_group.github_eragon_xyz_org_members (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "cloudflare_record.ssh_eragon_xyz (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "cloudflare_record.kibana_eragon_xyz (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "cloudflare_access_policy.es_policy (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "cloudflare_access_application.kibana_eragon_xyz (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "cloudflare_access_policy.kibana_policy (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "cloudflare_access_group.github_eragon_xyz_org_members" (*terraform.NodeApplyableResourceInstance) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "cloudflare_teams_account.eragon (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "cloudflare_access_identity_provider.github_eragon_xyz_oauth (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "cloudflare_record.es_eragon_xyz (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "cloudflare_record.aaaa_eragon_xyz (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "data.cloudflare_zone.eragon_xyz (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "cloudflare_access_application.es_eragon_xyz (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.152-0700 [DEBUG] ProviderTransformer: "cloudflare_zone_dnssec.eragon_xyz (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "cloudflare_access_group.github_eragon_xyz_org_members (expand)" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "var.account_id" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "cloudflare_access_group.github_eragon_xyz_org_members" references: [var.account_id cloudflare_access_identity_provider.github_eragon_xyz_oauth (expand) cloudflare_access_identity_provider.github_eragon_xyz_oauth (expand)]
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "cloudflare_record.ssh_eragon_xyz (expand)" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "cloudflare_teams_account.eragon (expand)" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "cloudflare_access_identity_provider.github_eragon_xyz_oauth (expand)" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "cloudflare_access_policy.es_policy (expand)" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "cloudflare_record.es_eragon_xyz (expand)" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "cloudflare_record.kibana_eragon_xyz (expand)" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "var.github_eragon_oauth_secret" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "cloudflare_record.postgres_eragon_xyz (expand)" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "cloudflare_record.aaaa_eragon_xyz (expand)" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "data.cloudflare_zone.eragon_xyz (expand)" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "var.cloudflare_email" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "var.cloudflare_api_key" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "cloudflare_access_application.es_eragon_xyz (expand)" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "cloudflare_access_application.kibana_eragon_xyz (expand)" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "cloudflare_zone_dnssec.eragon_xyz (expand)" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "cloudflare_access_policy.kibana_policy (expand)" references: []
2022-04-24T17:05:28.153-0700 [DEBUG] ReferenceTransformer: "provider[\"registry.terraform.io/cloudflare/cloudflare\"]" references: [var.cloudflare_api_key var.cloudflare_email]
2022-04-24T17:05:28.154-0700 [DEBUG] pruneUnusedNodes: cloudflare_record.es_eragon_xyz (expand) is no longer needed, removing
2022-04-24T17:05:28.154-0700 [DEBUG] pruneUnusedNodes: cloudflare_record.kibana_eragon_xyz (expand) is no longer needed, removing
2022-04-24T17:05:28.154-0700 [DEBUG] pruneUnusedNodes: cloudflare_record.postgres_eragon_xyz (expand) is no longer needed, removing
2022-04-24T17:05:28.154-0700 [DEBUG] pruneUnusedNodes: cloudflare_access_application.es_eragon_xyz (expand) is no longer needed, removing
2022-04-24T17:05:28.154-0700 [DEBUG] pruneUnusedNodes: cloudflare_access_application.kibana_eragon_xyz (expand) is no longer needed, removing
2022-04-24T17:05:28.154-0700 [DEBUG] pruneUnusedNodes: cloudflare_zone_dnssec.eragon_xyz (expand) is no longer needed, removing
2022-04-24T17:05:28.154-0700 [DEBUG] pruneUnusedNodes: cloudflare_access_policy.kibana_policy (expand) is no longer needed, removing
2022-04-24T17:05:28.154-0700 [DEBUG] pruneUnusedNodes: cloudflare_record.aaaa_eragon_xyz (expand) is no longer needed, removing
2022-04-24T17:05:28.154-0700 [DEBUG] pruneUnusedNodes: data.cloudflare_zone.eragon_xyz (expand) is no longer needed, removing
2022-04-24T17:05:28.154-0700 [DEBUG] pruneUnusedNodes: cloudflare_access_policy.es_policy (expand) is no longer needed, removing
2022-04-24T17:05:28.154-0700 [DEBUG] pruneUnusedNodes: cloudflare_teams_account.eragon (expand) is no longer needed, removing
2022-04-24T17:05:28.154-0700 [DEBUG] pruneUnusedNodes: cloudflare_record.ssh_eragon_xyz (expand) is no longer needed, removing
2022-04-24T17:05:28.154-0700 [DEBUG] Starting graph walk: walkApply
2022-04-24T17:05:28.155-0700 [DEBUG] created provider logger: level=debug
2022-04-24T17:05:28.155-0700 [INFO]  provider: configuring client automatic mTLS
2022-04-24T17:05:28.173-0700 [DEBUG] provider: starting plugin: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_arm64/terraform-provider-cloudflare_v3.13.0 args=[.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_arm64/terraform-provider-cloudflare_v3.13.0]
2022-04-24T17:05:28.175-0700 [DEBUG] provider: plugin started: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_arm64/terraform-provider-cloudflare_v3.13.0 pid=27461
2022-04-24T17:05:28.175-0700 [DEBUG] provider: waiting for RPC address: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_arm64/terraform-provider-cloudflare_v3.13.0
2022-04-24T17:05:28.183-0700 [INFO]  provider.terraform-provider-cloudflare_v3.13.0: configuring server automatic mTLS: timestamp=2022-04-24T17:05:28.183-0700
2022-04-24T17:05:28.195-0700 [DEBUG] provider: using plugin: version=5
2022-04-24T17:05:28.195-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0: plugin address: address=/var/folders/1c/lmn_0jyn0zgbtr2416y156n40000gn/T/plugin748331997 network=unix timestamp=2022-04-24T17:05:28.195-0700
2022-04-24T17:05:28.218-0700 [WARN]  ValidateProviderConfig from "provider[\"registry.terraform.io/cloudflare/cloudflare\"]" changed the config value, but that value is unused
2022-04-24T17:05:28.219-0700 [INFO]  provider.terraform-provider-cloudflare_v3.13.0: 2022/04/24 17:05:28 [INFO] Cloudflare Client configured for user: colemackenzie1@gmail.com: timestamp=2022-04-24T17:05:28.219-0700
cloudflare_access_group.github_eragon_xyz_org_members: Modifying... [id=0164a4ca-2bc2-415d-aad4-adf81ff345bf]
2022-04-24T17:05:28.223-0700 [INFO]  Starting apply for cloudflare_access_group.github_eragon_xyz_org_members
2022-04-24T17:05:28.223-0700 [DEBUG] cloudflare_access_group.github_eragon_xyz_org_members: applying the planned Update change
2022-04-24T17:05:28.224-0700 [INFO]  provider.terraform-provider-cloudflare_v3.13.0: 2022/04/24 17:05:28 [DEBUG] Updating Cloudflare Access Group from struct: {ID:0164a4ca-2bc2-415d-aad4-adf81ff345bf CreatedAt:<nil> UpdatedAt:<nil> Name:github-eragon-xyz-org-members Include:[] Exclude:[] Require:[]}: timestamp=2022-04-24T17:05:28.224-0700
2022-04-24T17:05:28.224-0700 [INFO]  provider.terraform-provider-cloudflare_v3.13.0: 2022/04/24 17:05:28 [DEBUG] Cloudflare API Request Details:
---[ REQUEST ]---------------------------------------
PUT /client/v4/accounts/f5686db3c4f5b3e38b8f15b0561a28a8/access/groups/0164a4ca-2bc2-415d-aad4-adf81ff345bf HTTP/1.1
Host: api.cloudflare.com
User-Agent: terraform/1.1.9 terraform-plugin-sdk/2.10.1 terraform-provider-cloudflare/3.13.0
Content-Length: 165
Content-Type: application/json
X-Auth-Email: [redacted]
X-Auth-Key: [redacted]
Accept-Encoding: gzip

{
 "id": "0164a4ca-2bc2-415d-aad4-adf81ff345bf",
 "created_at": null,
 "updated_at": null,
 "name": "github-eragon-xyz-org-members",
 "include": null,
 "exclude": null,
 "require": null
}
-----------------------------------------------------: timestamp=2022-04-24T17:05:28.224-0700
2022-04-24T17:05:28.721-0700 [INFO]  provider.terraform-provider-cloudflare_v3.13.0: 2022/04/24 17:05:28 [DEBUG] Cloudflare API Response Details:
---[ RESPONSE ]--------------------------------------
HTTP/2.0 200 OK
Cf-Cache-Status: DYNAMIC
Cf-Ray: 7012d3c3fc80682c-SEA
Content-Security-Policy: frame-ancestors 'none'; default-src https: 'unsafe-inline'
Content-Type: application/json; charset=UTF-8
Date: Mon, 25 Apr 2022 00:05:28 GMT
Expect-Ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
Server: cloudflare
Set-Cookie: __cflb=0H28vgHxwvgAQtjUGU4vq74ZFe3sNVUZdg65wCqgckd; SameSite=Lax; path=/; expires=Mon, 25-Apr-22 02:35:29 GMT; HttpOnly
Set-Cookie: __cfruid=572f84e8795478f65ef37e3a40815a7e12b11099-1650845128; path=/; domain=.api.cloudflare.com; HttpOnly; Secure; SameSite=None
Strict-Transport-Security: max-age=31536000; includeSubDomains
Vary: Accept-Encoding
X-Content-Type-Options: nosniff
X-Envoy-Upstream-Service-Time: 355
X-Frame-Options: DENY
X-Xss-Protection: 1; mode=block

{
  "result": {
    "id": "0164a4ca-2bc2-415d-aad4-adf81ff345bf",
    "name": "github-eragon-xyz-org-members",
    "uid": "0164a4ca-2bc2-415d-aad4-adf81ff345bf",
    "include": [],
    "require": [],
    "exclude": [],
    "created_at": "2022-04-24T17:24:24Z",
    "updated_at": "2022-04-25T00:05:28Z"
  },
  "success": true,
  "errors": [],
  "messages": []
}

-----------------------------------------------------: timestamp=2022-04-24T17:05:28.720-0700
2022-04-24T17:05:28.722-0700 [INFO]  provider.terraform-provider-cloudflare_v3.13.0: 2022/04/24 17:05:28 [DEBUG] Cloudflare API Request Details:
---[ REQUEST ]---------------------------------------
GET /client/v4/accounts/f5686db3c4f5b3e38b8f15b0561a28a8/access/groups/0164a4ca-2bc2-415d-aad4-adf81ff345bf HTTP/1.1
Host: api.cloudflare.com
User-Agent: terraform/1.1.9 terraform-plugin-sdk/2.10.1 terraform-provider-cloudflare/3.13.0
Content-Type: application/json
X-Auth-Email: [redacted]
X-Auth-Key: [redacted]
Accept-Encoding: gzip


-----------------------------------------------------: timestamp=2022-04-24T17:05:28.722-0700
2022-04-24T17:05:28.830-0700 [INFO]  provider.terraform-provider-cloudflare_v3.13.0: 2022/04/24 17:05:28 [DEBUG] Cloudflare API Response Details:
---[ RESPONSE ]--------------------------------------
HTTP/2.0 200 OK
Cf-Cache-Status: DYNAMIC
Cf-Ray: 7012d3c708a8dc59-SEA
Content-Security-Policy: frame-ancestors 'none'; default-src https: 'unsafe-inline'
Content-Type: application/json; charset=UTF-8
Date: Mon, 25 Apr 2022 00:05:28 GMT
Expect-Ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
Server: cloudflare
Set-Cookie: __cflb=0H28vgHxwvgAQtjUGU4vq74ZFe3sNVUZdg65wCqgckd; SameSite=Lax; path=/; expires=Mon, 25-Apr-22 02:35:29 GMT; HttpOnly
Set-Cookie: __cfruid=572f84e8795478f65ef37e3a40815a7e12b11099-1650845128; path=/; domain=.api.cloudflare.com; HttpOnly; Secure; SameSite=None
Strict-Transport-Security: max-age=31536000; includeSubDomains
Vary: Accept-Encoding
X-Content-Type-Options: nosniff
X-Envoy-Upstream-Service-Time: 10
X-Frame-Options: DENY
X-Xss-Protection: 1; mode=block

{
  "result": {
    "id": "0164a4ca-2bc2-415d-aad4-adf81ff345bf",
    "name": "github-eragon-xyz-org-members",
    "uid": "0164a4ca-2bc2-415d-aad4-adf81ff345bf",
    "include": [],
    "require": [],
    "exclude": [],
    "created_at": "2022-04-24T17:24:24Z",
    "updated_at": "2022-04-25T00:05:28Z"
  },
  "success": true,
  "errors": [],
  "messages": []
}

-----------------------------------------------------: timestamp=2022-04-24T17:05:28.829-0700
2022-04-24T17:05:28.832-0700 [WARN]  Provider "provider[\"registry.terraform.io/cloudflare/cloudflare\"]" produced an unexpected new value for cloudflare_access_group.github_eragon_xyz_org_members, but we are tolerating it because it is using the legacy plugin SDK.
    The following problems may be the cause of any confusing errors from downstream operations:
      - .include: block count changed from 1 to 0
      - .require: block count changed from 1 to 0
cloudflare_access_group.github_eragon_xyz_org_members: Modifications complete after 1s [id=0164a4ca-2bc2-415d-aad4-adf81ff345bf]
2022-04-24T17:05:28.835-0700 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = transport is closing"
2022-04-24T17:05:28.837-0700 [DEBUG] provider: plugin process exited: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_arm64/terraform-provider-cloudflare_v3.13.0 pid=27461
2022-04-24T17:05:28.837-0700 [DEBUG] provider: plugin exited
```

### Panic output

_No response_

### Expected output

The `include` and `require` blocks to be included in the API request

### Actual output

`include` and `require` blocks were not included in the PUT request for modifying the group inclusion rules.

### Steps to reproduce

1. Create Zero Trust Teams Account
2. Add GitHub as an identity provider
3. Create a group to include all members of the GitHub org


### Additional factoids

including other `include` or `require` directives configures them accordingly in the API

```hcl
require {
    login_method = [cloudflare_access_identity_provider.github_eragon_xyz_oauth.id]
}
```

### References

_No response_

After adding the specified `include` directive to the group in the UI and then trying to get terraform to parse the resource with the `github-organization` block, it results in a panic and cloudflare provider plugin crashing.

```bash
-----------------------------------------------------: timestamp=2022-04-24T17:29:19.408-0700
2022-04-24T17:29:19.660-0700 [INFO]  provider.terraform-provider-cloudflare_v3.13.0: 2022/04/24 17:29:19 [DEBUG] Cloudflare API Request Details:
---[ REQUEST ]---------------------------------------
GET /client/v4/accounts/f5686db3c4f5b3e38b8f15b0561a28a8/access/groups/0164a4ca-2bc2-415d-aad4-adf81ff345bf HTTP/1.1
Host: api.cloudflare.com
User-Agent: terraform/1.1.9 terraform-plugin-sdk/2.10.1 terraform-provider-cloudflare/3.13.0
Content-Type: application/json
X-Auth-Email: [redacted]
X-Auth-Key: [redacted]
Accept-Encoding: gzip


-----------------------------------------------------: timestamp=2022-04-24T17:29:19.659-0700
2022-04-24T17:29:19.776-0700 [INFO]  provider.terraform-provider-cloudflare_v3.13.0: 2022/04/24 17:29:19 [DEBUG] Cloudflare API Response Details:
---[ RESPONSE ]--------------------------------------
HTTP/2.0 200 OK
Cf-Cache-Status: DYNAMIC
Cf-Ray: 7012f6b66aa0e376-SEA
Content-Security-Policy: frame-ancestors 'none'; default-src https: 'unsafe-inline'
Content-Type: application/json; charset=UTF-8
Date: Mon, 25 Apr 2022 00:29:19 GMT
Expect-Ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
Server: cloudflare
Set-Cookie: __cflb=0H28vgHxwvgAQtjUGU4vq74ZFe3sNVUZL7uqEtcomW5; SameSite=Lax; path=/; expires=Mon, 25-Apr-22 02:59:20 GMT; HttpOnly
Set-Cookie: __cfruid=9048edac5f1159ba7916eeaf33d260346c8a061f-1650846559; path=/; domain=.api.cloudflare.com; HttpOnly; Secure; SameSite=None
Strict-Transport-Security: max-age=31536000; includeSubDomains
Vary: Accept-Encoding
X-Content-Type-Options: nosniff
X-Envoy-Upstream-Service-Time: 8
X-Frame-Options: DENY
X-Xss-Protection: 1; mode=block

{
  "result": {
    "id": "0164a4ca-2bc2-415d-aad4-adf81ff345bf",
    "name": "github-eragon-xyz-org-members",
    "uid": "0164a4ca-2bc2-415d-aad4-adf81ff345bf",
    "include": [
      {
        "login_method": {
          "id": "1541f135-e878-450f-816f-be39d33e18c5"
        }
      },
      {
        "github-organization": {
          "id": "88610774",
          "name": "eragon-xyz",
          "identity_provider_id": "1541f135-e878-450f-816f-be39d33e18c5"
        }
      }
    ],
    "require": [
      {
        "login_method": {
          "id": "1541f135-e878-450f-816f-be39d33e18c5"
        }
      }
    ],
    "exclude": [],
    "created_at": "2022-04-24T17:24:24Z",
    "updated_at": "2022-04-25T00:28:10Z"
  },
  "success": true,
  "errors": [],
  "messages": []
}

-----------------------------------------------------: timestamp=2022-04-24T17:29:19.775-0700
2022-04-24T17:29:19.779-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0: panic: interface conversion: interface {} is nil, not string
2022-04-24T17:29:19.779-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0:
2022-04-24T17:29:19.779-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0: goroutine 149 [running]:
2022-04-24T17:29:19.779-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0: github.com/cloudflare/terraform-provider-cloudflare/cloudflare.TransformAccessGroupForSchema({0x14000116f80?, 0x2, 0x7?})
2022-04-24T17:29:19.779-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0:    github.com/cloudflare/terraform-provider-cloudflare/cloudflare/resource_cloudflare_access_group.go:422 +0x34b8
2022-04-24T17:29:19.781-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0: github.com/cloudflare/terraform-provider-cloudflare/cloudflare.resourceCloudflareAccessGroupRead(0x140005ed700, {0x1036a13c0?, 0x14000216540})
2022-04-24T17:29:19.781-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0:    github.com/cloudflare/terraform-provider-cloudflare/cloudflare/resource_cloudflare_access_group.go:60 +0x6c4
2022-04-24T17:29:19.782-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0: github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0x1036afe20?, {0x1036afe20?, 0x1400012d590?}, 0xd?, {0x1036a13c0?, 0x14000216540?})
2022-04-24T17:29:19.782-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0:    github.com/hashicorp/terraform-plugin-sdk/v2@v2.14.0/helper/schema/resource.go:712 +0x138
2022-04-24T17:29:19.782-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0: github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0x140003d6fc0, {0x1036afe20, 0x1400012d590}, 0x14000147e10, {0x1036a13c0, 0x14000216540})
2022-04-24T17:29:19.782-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0:    github.com/hashicorp/terraform-plugin-sdk/v2@v2.14.0/helper/schema/resource.go:1015 +0x46c
2022-04-24T17:29:19.782-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0: github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0x140003b65e8, {0x1036afd78?, 0x14000117a40?}, 0x14000117ac0)
2022-04-24T17:29:19.784-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0:    github.com/hashicorp/terraform-plugin-sdk/v2@v2.14.0/helper/schema/grpc_provider.go:613 +0x40c
2022-04-24T17:29:19.784-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0: github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ReadResource(0x1400037c820, {0x1036afe20?, 0x1400012c840?}, 0x140005a48a0)
2022-04-24T17:29:19.784-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0:    github.com/hashicorp/terraform-plugin-go@v0.9.0/tfprotov5/tf5server/server.go:746 +0x3b8
2022-04-24T17:29:19.784-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0: github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadResource_Handler({0x103673d20?, 0x1400037c820}, {0x1036afe20, 0x1400012c840}, 0x140005a4840, 0x0)
2022-04-24T17:29:19.784-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0:    github.com/hashicorp/terraform-plugin-go@v0.9.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:349 +0x174
2022-04-24T17:29:19.784-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0: google.golang.org/grpc.(*Server).processUnaryRPC(0x1400033ea80, {0x1036b2870, 0x140001024e0}, 0x14000698fc0, 0x140004373b0, 0x103b8d390, 0x0)
2022-04-24T17:29:19.784-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0:    google.golang.org/grpc@v1.45.0/server.go:1282 +0xb3c
2022-04-24T17:29:19.784-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0: google.golang.org/grpc.(*Server).handleStream(0x1400033ea80, {0x1036b2870, 0x140001024e0}, 0x14000698fc0, 0x0)
2022-04-24T17:29:19.784-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0:    google.golang.org/grpc@v1.45.0/server.go:1619 +0x840
2022-04-24T17:29:19.784-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0: google.golang.org/grpc.(*Server).serveStreams.func1.2()
2022-04-24T17:29:19.784-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0:    google.golang.org/grpc@v1.45.0/server.go:921 +0x88
2022-04-24T17:29:19.784-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0: created by google.golang.org/grpc.(*Server).serveStreams.func1
2022-04-24T17:29:19.784-0700 [DEBUG] provider.terraform-provider-cloudflare_v3.13.0:    google.golang.org/grpc@v1.45.0/server.go:919 +0x298
2022-04-24T17:29:19.785-0700 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = transport is closing"
2022-04-24T17:29:19.785-0700 [DEBUG] provider: plugin process exited: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_arm64/terraform-provider-cloudflare_v3.13.0 pid=27989 error="exit status 2"
2022-04-24T17:29:19.785-0700 [ERROR] plugin.(*GRPCProvider).ReadResource: error="rpc error: code = Unavailable desc = transport is closing"
2022-04-24T17:29:19.785-0700 [ERROR] plugin.(*GRPCProvider).ReadResource: error="rpc error: code = Unavailable desc = transport is closing"
2022-04-24T17:29:19.785-0700 [ERROR] vertex "cloudflare_access_group.github_eragon_xyz_org_members" error: Plugin did not respond
2022-04-24T17:29:19.785-0700 [ERROR] plugin.(*GRPCProvider).ReadResource: error="rpc error: code = Unavailable desc = transport is closing"
2022-04-24T17:29:19.785-0700 [ERROR] vertex "cloudflare_access_group.github_eragon_xyz_org_members" error: Plugin did not respond
2022-04-24T17:29:19.785-0700 [ERROR] vertex "cloudflare_access_group.github_eragon_xyz_org_members (expand)" error: Plugin did not respond
2022-04-24T17:29:19.785-0700 [ERROR] vertex "cloudflare_teams_account.eragon" error: Plugin did not respond
2022-04-24T17:29:19.785-0700 [ERROR] vertex "cloudflare_zone_dnssec.eragon_xyz" error: Plugin did not respond
2022-04-24T17:29:19.785-0700 [ERROR] vertex "cloudflare_zone_dnssec.eragon_xyz" error: Plugin did not respond
2022-04-24T17:29:19.785-0700 [ERROR] vertex "cloudflare_teams_account.eragon" error: Plugin did not respond
2022-04-24T17:29:19.785-0700 [ERROR] vertex "cloudflare_zone_dnssec.eragon_xyz (expand)" error: Plugin did not respond
2022-04-24T17:29:19.785-0700 [ERROR] vertex "cloudflare_teams_account.eragon (expand)" error: Plugin did not respond
```

--- a/.changelog/1589.txt
+++ b/.changelog/1589.txt
@@ -1,0 +1,3 @@
+```release-note:bug
+resource/cloudflare_access_group: allow github access groups to be created without a list of teams
+```

--- a/cloudflare/resource_cloudflare_access_group.go
+++ b/cloudflare/resource_cloudflare_access_group.go
@@ -90,9 +90,7 @@ func resourceCloudflareAccessGroupCreate(d *schema.ResourceData, meta interface{
 	}
 
 	d.SetId(accessGroup.ID)
-	resourceCloudflareAccessGroupRead(d, meta)
-
-	return nil
+	return resourceCloudflareAccessGroupRead(d, meta)
 }
 
 func resourceCloudflareAccessGroupUpdate(d *schema.ResourceData, meta interface{}) error {
@@ -248,14 +246,25 @@ func BuildAccessGroupCondition(options map[string]interface{}) []interface{} {
 		} else if accessGroupType == "github" {
 			for _, v := range values.([]interface{}) {
 				githubCfg := v.(map[string]interface{})
-				for _, team := range githubCfg["teams"].([]interface{}) {
+				if len(githubCfg["teams"].([]interface{})) > 0 {
+					for _, team := range githubCfg["teams"].([]interface{}) {
+						group = append(group, cloudflare.AccessGroupGitHub{GitHubOrganization: struct {
+							Name               string `json:"name"`
+							Team               string `json:"team,omitempty"`
+							IdentityProviderID string `json:"identity_provider_id"`
+						}{
+							Name:               githubCfg["name"].(string),
+							Team:               team.(string),
+							IdentityProviderID: githubCfg["identity_provider_id"].(string),
+						}})
+					}
+				} else {
 					group = append(group, cloudflare.AccessGroupGitHub{GitHubOrganization: struct {
 						Name               string `json:"name"`
 						Team               string `json:"team,omitempty"`
 						IdentityProviderID string `json:"identity_provider_id"`
 					}{
 						Name:               githubCfg["name"].(string),
-						Team:               team.(string),
 						IdentityProviderID: githubCfg["identity_provider_id"].(string),
 					}})
 				}
@@ -419,7 +428,9 @@ func TransformAccessGroupForSchema(accessGroup []interface{}) []map[string]inter
 				githubCfg := groupValue.(map[string]interface{})
 				githubID = githubCfg["identity_provider_id"].(string)
 				githubName = githubCfg["name"].(string)
-				githubTeams = append(githubTeams, githubCfg["team"].(string))
+				if v, ok := githubCfg["team"]; ok {
+					githubTeams = append(githubTeams, v.(string))
+				}
 			case "azureAD":
 				azureCfg := groupValue.(map[string]interface{})
 				azureID = azureCfg["identity_provider_id"].(string)
@@ -510,7 +521,7 @@ func TransformAccessGroupForSchema(accessGroup []interface{}) []map[string]inter
 		})
 	}
 
-	if len(githubTeams) > 0 && githubID != "" && githubName != "" {
+	if githubID != "" && githubName != "" {
 		data = append(data, map[string]interface{}{
 			"github": []interface{}{
 				map[string]interface{}{

